topics/scikit-learn/index.md

```diff
@@ -0,0 +1,13 @@
+---
+aliases: sklearn
+created_by: David Cournapeau, The scikit-learn developers
+display_name: scikit-learn
+github_url: https://github.com/scikit-learn/scikit-learn
+released: January 05, 2010
+short_description: scikit-learn is a Python module for machine learning.
+topic: scikit-learn
+related: scikit, python
+url: https://github.com/scikit-learn/scikit-learn/releases
+wikipedia_url: https://en.wikipedia.org/wiki/Scikit-learn
+---
+scikit-learn is a widely used [de facto standard](https://en.wikipedia.org/wiki/De_facto_standard) Python module for classic machine learning built on top of SciPy.
```

Comment:
Please either use the unlinked word "standard" or just remove "de facto standard" entirely. We're aware of what a de facto standard is, we just don't agree it's the best wording. Thanks!